packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -41,6 +47,13 @@ const lightModeNeutralOutlineRest: DesignSystemResolver<string> = neutralOutline
     (): string => white
 );
 
+const darkModeNneutralFocus: DesignSystemResolver<string> = neutralFocus(
```

Comment:
Can we call these `themeDark` and `themeLight` instead to match the style names below. Those names were meant to be based on the setting for the content of the slides rather than the overall UI color, which we have reserved "mode" for.